doc/modules/neural_networks_supervised.rst

```diff
@@ -118,19 +110,18 @@ classification, it minimizes the Cross-Entropy loss function, giving a vector
 of probability estimates :math:`P(y|x)` per sample :math:`x`::
 
     >>> clf.predict_proba([[2., 2.], [1., 2.]])  # doctest: +ELLIPSIS
-    array([[ 0.,  1.],
-           [ 0.,  1.]])
+    array([[  1.96718015e-04,   9.99803282e-01],
```

Comment:
you can use `ELLIPSIS` instead? Refer [this](https://pymotw.com/2/doctest/#handling-unpredictable-output)